lib/haxe/src/org/apache/thrift/transport/TFileStream.hx

```diff
@@ -24,6 +24,9 @@ import haxe.io.BytesBuffer;
 import haxe.io.Input;
 import haxe.io.Output;
 
+#if js
+import js.node.Fs;
+#end
 
```

Comment:
We have basically two dource files in one. Maybe we should refactor this .later on.